src/feat/signal-distort.h

```diff
@@ -59,19 +63,35 @@ struct XvectorPerturbOptions {
     opts->Register("noise-egs", &noise_egs, "If supplied, the additive noise is added to input signal.");
     opts->Register("rand_distort", &rand_distort, "If true, the signal is slightly changes"
                    "using some designed FIR filter with no zeros.");
+    opts->Register("add-noise", &add_noise, "specify a file contains some noise egs");
```

Comment:
change the definition! e.g. Noise rspecifier for additive noises, if nonempty, the additive noise randomly selected and added to input egs.